src/assessments/readability/subheadingDistributionTooLongAssessment.js

```diff
@@ -38,27 +80,38 @@ class SubheadingsDistributionTooLong extends Assessment {
 	 *
 	 * @param {Paper} paper The paper to use for the assessment.
 	 * @param {Researcher} researcher The researcher used for calling research.
-	 * @param {object} i18n The object used for translations.
+	 * @param {Object} i18n The object used for translations.
 	 *
 	 * @returns {AssessmentResult} The assessment result.
 	 */
 	getResult( paper, researcher, i18n ) {
-		let subheadingTextsLength = researcher.getResearch( "getSubheadingTextLengths" );
-		subheadingTextsLength = subheadingTextsLength.sort(
-			function( a, b ) {
-				return b.wordCount - a.wordCount;
-			}
-		);
+		this._subheadingTextsLength = researcher.getResearch( "getSubheadingTextLengths" );
```

Comment:
`const subheadingTextsLength = researcher.getResearch( "getSubheadingTextLengths" );`

Reply:
I like to keep the research results as an attribute of the class, this way I can reach them from other functions within this class.